modules/rosa-hcp-vpc-terraform.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-hcp-vpc-terraform_{context}"]
+= Creating an AWS VPC using Terraform
+
+Terraform is a tool that allows you to create various resources using an established template. The following process uses the default options as required to create a {hcp} for ROSA cluster. For more information about using Terraform, see the additional resources.
+
+.Prerequisites
+
+* You have installed Terraform on your machine.
+
+.Procedure
+
+. In your CLI, create a directory for your terraform files, and then navigate to that directory.
++
+[source,terminal]
+----
+$ mkdir hypershift-tf && cd hypershift-tf
+----
+
+. Download the VPC setup terraform file.
+//This is Will's personal one, so we may need to create one for GA
```

Comment:
```suggestion
```

Updated to use a more generalized template, we will _not_ be providing a terraform option for GA (as far as I'm aware)

Reply:
Could documentation create a template for the sake of standardizing our docs? I'm not totally certain of the logistics of that, but it could be very helpful!